### PR TITLE
fix: mobile styles for kiva card updated

### PR DESCRIPTION
**Pull request by georgebs**
**Ticket:**

[https://kiva.atlassian.net/jira/software/projects/CORE/boards/248?selectedIssue=CORE-376](https://kiva.atlassian.net/jira/software/projects/CORE/boards/248?selectedIssue=CORE-376)

**Steps to test:**

1. Login
2. Go to http://localhost:8888/checkout
3. Resize the screen to 320px
4. Open the "Have a Kiva card ?" option
5. Each one of the 3 elements should be in it's own row like is shown in the figma design [https://www.figma.com/file/KHkcWqBeHNuPIxLifrkUkg/Checkout-2021---Kiva-Classic-%5BJoice.Aug.2021%5D?node-id=665%3A2028](https://www.figma.com/file/KHkcWqBeHNuPIxLifrkUkg/Checkout-2021---Kiva-Classic-%5BJoice.Aug.2021%5D?node-id=665%3A2028)

### CODE REVIEW

File: src/components/Checkout/KivaCardRedemption.vue [hunk 1]
```diff
@@ -22,9 +22,9 @@
 				v-show="open"
 				class="accordion-info row small-12 columns"
 			>
-				<div class="small-9 small-offset-3 large-10 large-offset-2 columns">
+				<div class="small-12 large-10 large-offset-2 columns">
```
Comment (mcstover) [inline src/components/Checkout/KivaCardRedemption.vue:25]: Could you please remove the `row` class and all `small-, large-, etc` classes which are rooted in Foundation. You'll need to replace them with the tailwind implementation of flex box. to update the layout of this section of the page.

**Review by mcstover — CHANGES_REQUESTED** [review]
We're trying to remove as much of the Foundation classes as possible when we work in these files, converting them to tailwinds classes.

File: src/components/Checkout/KivaCardRedemption.vue (hunk 1 above)
Comment (georgebs) [reply src/components/Checkout/KivaCardRedemption.vue:25]: Oh I see, I didn't know that, but sure! will do 👍

Comment (georgebs) [reply src/components/Checkout/KivaCardRedemption.vue:25]: A new ticket will be opened to change those classes as it's has been discussed

**Review by mcstover — APPROVED** [review]

**Review by roger-in-kiva — APPROVED** [review]

**Pull request merged**